pkg/querier/ingester_querier.go

```diff
@@ -269,7 +269,7 @@ func (q *IngesterQuerier) TailersCount(ctx context.Context) ([]uint32, error) {
 		return nil, err
 	}
 
-	counts := make([]uint32, len(responses))
```

Comment:
this seems like a good change, AFAICT atm we could be allocating `# of ingesters * 2` uint32's since we could have a response from every ingester